markdown/articles/2021/08/22/Non-blue-fair-decks.md

```diff
@@ -0,0 +1,429 @@
+---
+authors: OtterLady
+banner: /assets/stinging-study.c21.jpg
+title: "Matchup Guides: Non Blue Fair Decks"
+---
+
+With this chapter we want to give you a brief overview over the most relevant matchups in the current meta. We try to keep everything short and easy to read.
+
+!If you want to skip to a certain matchup use STRG+F!
+
```

Comment:
Maybe lead with a description of how each entry is laid out?

eg:

Opposing archetype name:
Gameplan: A short summary of how we should approach the matchup.
Cards to keep in mind: The main relevant cards both pre-boarding and post-boarding in that the opposing deck may have.
Gameplay: What sort of hands you should look to keep and how you should try to play (build depending). This is looked at both before the "combo turn" and after a successful DD resolution.

Reply:
added